Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -112,6 +116,9 @@ class BrowserViewController: UIViewController, BrowserToolbarDelegate, TabManage
                 if let progress = change[NSKeyValueChangeNewKey] as Float? {
                     toolbar.updateProgressBar(progress)
                 }
+
+                toolbar.updateBackStatus(object.canGoBack)
+                toolbar.updateFowardStatus(object.canGoForward)
```

Comment:
I remember thinking the same thing, then trying it via webViewDidStartLoading and having some issue, then deciding that since UIKit optimizes this anyway and it's not expensive unless the state actually changes (I *believe*).

I'll dive back in and see if I remember/encounter whatever the issue was and update accordingly. Thanks.